### ISSUE TITLE
Eventually move to JuliaPOMDPs and rename to Crux.jl

### ISSUE THREAD

**Issue by mossr** [ancorso/Crux#1]
Mykel and I agree this fits nicely into JuliaPOMDPs (as opposed to sisl or here). No rush, of course—we'll want to stress it and let it mature a bit before we bring it over. Thoughts?

**Comment by ancorso** [comment]
Yeah sounds good! The main thing I want to do before making it more public is write up some documentation for the API as well as a github.io page like https://juliapomdp.github.io/POMDPs.jl/stable/.

**Comment by mossr** [comment]
[Documenter.jl](https://github.com/JuliaDocs/Documenter.jl) is excellent for this (that's what POMDPs.jl is using, and I've used it for POMDPStressTesting.jl as well).

https://sisl.github.io/POMDPStressTesting.jl/dev/

**Comment by mossr** [comment]
Do you mind if I move Crux and POMDPGym to the sisl Github sometime next week? I'd also like to register them with General so I can use them in my POMDPs.jl Julia Academy course, but understand the hesitation mentioned above (i.e. documentation before public release)

**Comment by mykelk** [comment]
Shall we go ahead and move to this to JuliaPOMDP @ancorso and @mossr ?
